projects/tuist/features/step_definitions/test.rb

```diff
@@ -1,47 +1,54 @@
 # frozen_string_literal: true
 
-require "xcodeproj"
+require 'xcodeproj'
```

Comment:
nit: I don't see a reason for the change of quotation marks? Makes it harder to follow the actual changes here.  In the other `step_definitions` files, we tend to use `"` instead of `'`

Reply:
@fortmarek ah sorry about that, I have rubocop installed globally which do these changes automatically. I'll revert them. As an improvement, it may be good to such rules for Tuist so the style is kept consistency across all Ruby files?